### PR TITLE
[2712][IMP] account_invoice_report_hls: switch to qweb

### PR DESCRIPTION
**Pull request by kanda999**
[2712](https://www.quartile.co/web?#id=2712&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: account_invoice_report_hls/report/invoice_delivery.xml [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+                <div class="col-12">
+                    <div style="border-bottom: 1px solid black;" />
+                </div>
+            </div> -->
+            <!-- <div class="row">
+                <div t-if="doc_type != 'invoice'" class="col-12 text-right">
+                    <strong t-field="o.invoice_id.date_invoice" t-options='{"widget": "date"}' />
+                </div>
+            </div> -->
+        </div>
```
Comment (yostashiro) [inline account_invoice_report_hls/report/invoice_delivery.xml]:
```suggestion
```
Any reason to still keep this part?

File: account_invoice_report_hls/report/invoice_delivery.xml [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                </h2>
+                <div class="row" style="padding-top: 20px;">
+                    <div class="col-6">
+                        <div>
+                            <address t-field="o.invoice_id.partner_id" t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}' />
+                        </div>
+                        <div style="padding-top: 60px;">
+                            <div>
+                                <p>
+                                    <div>平素は格別のご高配に賜り熱く御礼申し上げます。</div>
```
Comment (yostashiro) [inline account_invoice_report_hls/report/invoice_delivery.xml]:
```suggestion
                                    <div>平素は格別のご高配に賜り厚く御礼申し上げます。</div>
```
We should ideally make this sentence configurable in company form, although it is not a must at this stage.

**Review by yostashiro — CHANGES_REQUESTED** [review]
We need to sort out the transition steps. With the current way of updating account_invoice_report_hls, the user will lose the option of printing the invoice with the currently available function (py3o) once we apply this change in their environment, which I think is a bit risky.

Can we do either:

1. update account_invoice_report_hls in the manner that does not affect the existing functions,

or

2. add a new module so that the new print function and the existing one can co-exist?

Once the client confirms that the new print function works fine in the production environment, then we will remove the existing function.

**Comment by kanda999** [comment]
@yostashiro 
I'll choose option 2, thinking about removing later.
> 2,add a new module so that the new print function and the existing one can co-exist

File: account_invoice_report_hls/report/invoice_delivery.xml (hunk 1 above)
Comment (kanda999) [reply account_invoice_report_hls/report/invoice_delivery.xml]: I thought I might need the header later , but I don't need one.

**Comment by kanda999** [comment]
In the end I chose option 2.

**Review by yostashiro — APPROVED** [review]
Made some changes so that you won't have to go through a wizard to print the QWeb report.

**Review by yostashiro — APPROVED** [review]

**Pull request merged**